### DISCUSSION TITLE
그림자 처럼 따라 다니게 하기

### DISCUSSION THREAD

**Discussion by KwonCheolHyeon** [naverz/zepeto-studio-kor#1211] [5. Q&A]
멀티 플레이 게임을 만들던 중 궁금하게 있어 질문 합니다.
술래잡기 게임을 만들고 있는데 AI가 플레이어를 따라가는 형식입니다. 
타겟 오브젝트가 플레이어를  CharacterShadow 처럼 계속 따라 다니게 하는 방법이 궁금합니다.

**Comment by zlmrzlo** [comment]
이 부분은 AI 영역인 것 같아요. navMeshAgent로 검색해서 정보를 찾으면 될 것 같습니다. 유튜브에 따라할 수 있는 것들이 많이 존재합니다.

**Reply by KwonCheolHyeon** [reply]
감사합니다

**Comment by YSHFM** [comment]
단순히 따라가게 한다면 transform.LookAt()으로 타겟을 바라보게 만든 후,
 forward방향으로 속도값과 deltatime값에 따라 이동시키게 할 수 있겠습니다만,
 애니메이션과 장애물, 여러 요소를 고려한다면 직접 정보를 찾으시면서 알맞는 정보로 직접 로직을 구성하시는 게 더 빠르실 것 같습니다~

**Reply by KwonCheolHyeon** [reply]
감사합니다
